### ISSUE TITLE
Cannot compile c_dy_respawn_naong.sp

### ISSUE THREAD

**Issue by superforno** [Daimyo21/Insurgency-dy-sourcemod#31]
Hello,
first of all thanks for sharing on GitHub your plugins!

I've a server running with latest sourcemod and your plugins, but now I've tryied to enable the mysql rankings on c_dy_respawn_naong.sp. 

As result I've got 4 errors in compiling:
(720) : error 017: undefined symbol "Event_GrenadeThorwn"
(741) : error 017: undefined symbol "Event_PlayerTeam"
(5023) : error 017: undefined symbol "Check_NearbyMedicsRevive"
(5194) : error 017: undefined symbol "Check_NearbyMedicsRevive"

Any hint? 
Thanks

**Comment by thecannons** [comment]
Hi Enrico,

Do you have the insurgencydy.inc file in your include folder (or make sure
the #includes at the top of the file exist in your include folder)?  Just
make sure you have all dependencies.

**Comment by superforno** [comment]
Hello! Thas't the includes part
`#include <sourcemod>
#include <sdktools>
#include <sdkhooks>
#include <insurgencydy>
#include <smlib>
#undef REQUIRE_EXTENSIONS
#include <cstrike>
#include <tf2>
#include <tf2_stocks>
#define REQUIRE_EXTENSIONS

//#include <navmesh>
//#include <insurgency>`

Each file is present in the "include" folder.
Could be maybe the ones commented out at the end?

**Comment by thecannons** [comment]
The latest script I support is c_dy_respawn_naong_ai_director so I might
have butchered the insurgencydy.inc file after some time.  I just don't
have the energy to go through it lately.

**Comment by superforno** [comment]
Ok, so, should I use the file you mentioned last, c_dy_respawn_naong_ai_director?

**Comment by thecannons** [comment]
You can try but it includes latest features that you may not like.

**Comment by superforno** [comment]
I'll try it.
Anyway seems that errors happens when enabling the rank system. Any hint on this?

Thanks!

**Comment by ozzygit** [comment]
Yeah, I've been asked to look at scoring from one of my players too. Because I haven't installed mySQL on my linux server so I'd have to do this for ranking I think, but worth doing.

There is much hard coded and this is what I've started removing too. If you wanted, perhaps we could work on it together?

________________________________

**Comment by superforno** [comment]
I'll be glad to work on it! I've already set up a mysql db for ranking, but I can't compile the plugin

**Comment by ozzygit** [comment]
Do you have any coding skills such as java, c#, c++ etc? In using the ai director version which means elite bots

Get Outlook for Android<https://aka.ms/ghei36>

________________________________

**Comment by superforno** [comment]
I've programmed mostly in php, so I can understand basic c++ scripts. 
I'm actually triyng the ai director too and I like it! So I think I'll edit that one...

**Comment by superforno** [comment]
PS: Interesting thing I've just noticed, with c_dy_respawn_naong_ai_director Medics functions (revive and heal) are not working anymore!

**Comment by ozzygit** [comment]
Just posting here saying kudos to Daimyo for all his hard work and all credit goes to Daimyo for being able to build on his work like he did for JBallou.
